README.md

```diff
@@ -0,0 +1,51 @@
+# ZoomTube
+
+View our [Design Doc](https://docs.google.com/document/d/12t7FDPDpTpHzrQBJ3T8qdmGKGrPi3lr2-UPmoc2ISHI).
```

Comment:
I just enabled link sharing to anyone with a Google email. The shared link is slightly different, not too sure if the link in the readme needs to be updated.
```suggestion
View our [Design Doc](https://docs.google.com/document/d/12t7FDPDpTpHzrQBJ3T8qdmGKGrPi3lr2-UPmoc2ISHI/edit?usp=sharing).
```